pkg/metal/types.go

```diff
@@ -77,8 +77,9 @@ const (
 	ShootExtensionTypeTokenIssuer = "tokenissuer"
 
 	// Shoot Annotations
-	ShootAnnotationCreatedBy   = "garden.sapcloud.io/createdBy"
-	ShootAnnotationPurpose     = "garden.sapcloud.io/purpose"
+	ShootAnnotationCreatedBy = "garden.sapcloud.io/createdBy"
```

Comment:
i think these annotations are different now

Reply:
These annotations were not used anywhere in our project. I also cleaned up the rest of the unused constants.